eng/tox/tox.ini

```diff
@@ -51,21 +51,23 @@ platform = linux: linux
            macos: darwin
            windows: win32
 passenv = *
+download=true
 requires=
   {[packaging]pkgs}
 setenv =
   SPHINX_APIDOC_OPTIONS=members,undoc-members,inherited-members
   PIP_EXTRA_INDEX_URL=https://pypi.python.org/simple
+  VIRTUALENV_WHEEL=0.37.0
+  VIRTUALENV_PIP=20.3.3
+  VIRTUALENV_SETUPTOOLS=60.7.0
 deps = {[base]deps}
 changedir = {toxinidir}
-install_command = python -m pip install {opts} {packages} --cache-dir {toxinidir}/../.tox_pip_cache_{envname}
-commands_pre=
-    {envbindir}/python -m pip install pip==20.3
+install_command = python -m pip install {opts} {packages} --cache-dir {toxinidir}/../.tox_pip_cache_{envname} --ignore-installed
```

Comment:
`--ignore-installed` is because on pypy3.7/pypy3.8, the _virtualenv_ pip is finding site-packages from the _global_ install. [This is what that looked like](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1342443&view=logs&j=563de66a-0637-56b5-a77c-8b9a24c478c0&t=b0c3bd39-e5d0-5f0d-f1d7-d5569ea832ef&l=2670).

![image](https://user-images.githubusercontent.com/45376673/152273257-749344ca-66ab-40b8-ace3-2b25e3c7c601.png)

Aside from this install addition, I have not been able to get that behavior to STOP. For clarity, on `pypy3.6.12` works JUST FINE regardless if I prepend the path using MY python script vs the default `pypy3` selected by UsePythonVersion task.

That means that the issue causing this weird behavior is native to pypy, and not specific to devops.